gitops/azure-devops/ManifestGeneration.md

```diff
@@ -83,18 +88,35 @@ In Azure DevOps:
   ![SELECT REPO TYPE](images/select-git-repo.png)
 
 6. Choose your high level definition repo:
-    1. Azure DevOps Repo Example: ![SELECT REPO NAME](images/select-ado-repo.png)
-    1. GitHub Repo Example:![SELECT REPO NAME](images/select-github-repo.png)
+    1. Azure DevOps Repo Example: 
```

Comment:
@drewby these changes seems unnecessary. If we could revert this along with the related images changes we should be good to go.